R/trans-numeric.r

```diff
@@ -16,17 +16,17 @@ atanh_trans <- function() {
   trans_new("atanh", "atanh", "tanh")
 }
 
-#' Box-Cox (modulus) power transformation.
+#' Modulus transformation.
```

Comment:
Document modulus and boxcox together?

Maybe include the equations in the description?